JSDemos/Demos/Scheduler/CustomTemplates/description.md

```diff
@@ -1 +1,7 @@
-This demo illustrates the rich collection of&nbsp;the appearance customization properties available as&nbsp;a&nbsp;part of&nbsp;the Scheduler component. The appearance of&nbsp;appointments and appointment tooltips are defined using custom templates. Properties for the appointment details form are set within the **appointmentFormOpening** event handler. Properties for the Scheduler are also modified: the **timelineDay** view is&nbsp;added to&nbsp;the _View Selector_ and the _All-day panel_ is&nbsp;hidden.
\ No newline at end of file
+This demo shows the following Scheduler properties that specify custom templates:
+
+* [appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#appointmentTemplate) / **views[]**.[appointmentTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#appointmentTemplate)
+
+* [appointmentTooltipTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#appointmentTooltipTemplate) / **views[]**.[appointmentTooltipTemplate](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/views/#appointmentTooltipTemplate).
+
+To customize the appointment details form, implement the [onAppointmentFormOpening](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentFormOpening) handler. In this demo, we use it to add custom fields.
```

Comment:
Let;s get rid of the first person plural here. Maybe something like that? 

In this demo, this handler adds custom fields to the appointment details form.